decidim-surveys/app/models/decidim/surveys/survey.rb

```diff
@@ -48,6 +48,10 @@ def open?
         false
       end
 
+      def clean_after_publish?
+        clean_after_publish
+      end
+
```

Comment:
I think this is unnecessary, booleans are automatically defined with a ? suffix

Reply:
OH OK. Thanks